src/Core/Domain/Currency/Exception/CurrencyConstraintException.php

```diff
@@ -72,16 +72,10 @@ class CurrencyConstraintException extends CurrencyException
      */
     const EMPTY_SYMBOL = 7;
 
-    /**
-     * Code used when trying to insert a currency referenced in CLDR database but ISO code and
-     * numeric ISO code are not matching the expected ones
-     */
-    const ISO_CODES_MISMATCH = 8;
```

Comment:
@jolelievre It's not a BC to remove a public constant ?

Reply:
No because it was introduced in 177, so we can still clean it for now

Reply:
Ok for me.